test-network/addOrg3/docker/docker-compose-org3.yaml

```diff
@@ -50,33 +50,3 @@ services:
       - 11051:11051
     networks:
       - test
-
```

Comment:
This CLI container is used by the adding an org to the channel tutorial. As a result, I think it should be left in to not break the existing tutorial, even if it is not used by the script. We can update the tutorial to use the new CLI container, and then remove this redundant container afterwards

Reply:
I was planning to update the tutorial immediately afterwards, but I guess there's no reason I can't push a PR now even before this is merged.

Reply:
Actually, you're right, there are more moving updates to the doc than I realized. Reintroduced the Org3cli container temporarily until we're able to update the tutorials.

Reply:
Tested locally, and ran through a bit of the Org3 tutorial and everything works. 

I think this is good enough to merge, and includes a lot of overdue clean up, once we have come to a decision on whether it is worth bringing back the Org3 container or not.

Reply:
Interestingly, the create channel tutorial actually already documents the correct way to set the anchor peers for each organization and avoids using a `fabric-tools` cli container. Maybe with that in mind we should just move away from the `fabric-tools` containers entirely and just require `jq` be installed as a prereq for the `test-network` at this point?

Reply:
I don't like the CLI container, but I also don't like dependencies for people just kicking the tires. My opinion is to keep CLI container so that people just kicking the tires with test-network, but not doing any config transactions, wouldn't require the jq dependency. It would only be required by people doing config transactions themselves, which is a whole different level of tire kicking which could have more dependencies. But let's get some other opinions as well.

Reply:
Discussed this with the crowd on today's fabric contributors meeting. Everyone present (that voiced an opinion) agreed that sticking with the CLI container for the automated scripts (`./network.sh createChannel` and `./addOrg3.sh up`) to set the anchor peers is ideal to keep the barrier to entry low for new users. For the tutorials that demonstrate running each command manually, we'll make sure all of those run on the local machine and avoid a CLI container. The original tutorial for creating a channel already does this while the adding an org to a channel tutorial still uses the org3cli container. I'll also add a note regarding the CLI container and why it's used in the scripts but not the manual steps.

Reply:
Sounds like a good plan.